native/cocos/renderer/GFXDeviceManager.h

```diff
@@ -36,6 +36,12 @@
 // #undef CC_USE_GLES3
 // #undef CC_USE_GLES2
 
+// arengine only supports gles2, arcore supports gles2 and gles3
+// setting the CC_USE_GLES3 off is needed while using CC_USE_AR_AUTO or CC_USE_AR_ENGINE
+#if CC_USE_AR_MODULE && (CC_USE_AR_AUTO || CC_USE_AR_ENGINE)
```

Comment:
What device only supports gles2?

Reply:
Currently on HuaWei Devices that supports AREngine cannot get normal camera image while use gles3